README.md

```diff
@@ -18,6 +18,9 @@ What are constructor functions and the `new` keyword? What is a prototype? Descr
 Your Answer:
 ```text
 
+An object constructor function is used to create an object that can be used many times without recreating the object each time. The 'new' operator calls the object constructor. Prototypes are a type of inheritance in Javascript. Prototypes are basically methods that are attached to an object after it is defined.
```

Comment:
Close.  Prototypes are objects, which contain properties and methods that are shared across objects.